### PR TITLE
Update readme for 2019 NARSC

### PR DESCRIPTION
**Pull request by weikang9009**
* update timelines for the workshop (8am-5pm) 
* update WK's website link

### CODE REVIEW

**Comment by sjsrey** [comment]
the branch we are working on is 2019-narsc https://github.com/sjsrey/pysalworkshop/tree/2019-narsc

**Comment by weikang9009** [comment]
Sorry. My mistake. Just changed the base branch.

File: README.md [hunk 1]
```diff
@@ -13,29 +13,32 @@ This repository contains the materials and instructions for the PySAL workshop
...
   * Global spatial autocorrelation
   * Local spatial autocorrelation
-* 15:45-16:15
+* 14:45-15:15
   * Coffee Break  
-* 16:15-17:30
+* 15:15-17:00
   * Spatial inequality dynamics
   * Regionalization and clustering
+  * Basic spatial regression
```
Comment (sjsrey) [inline README.md]: Do you want to add the content for this?

Comment (weikang9009) [reply README.md]: I just deleted it.

**Pull request merged**